### PR TITLE
Variants project: improve readme

### PR DESCRIPTION
**Pull request by phkuo**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Improve the readme for the Variants project

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/variants/README.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
-This project creates variants of a given theme, which are intended for use with the Customizer component in office-ui-fabric-react. These subthemes allow you to highlight or de-emphasize sections of the page with colors that still match the current theme.
+This project provides helper functions to create variants of a given theme, which are intended for use with the [Customizer](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/utilities/src/Customizer.tsx) component. Each function takes in an [IPartialTheme](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/interfaces/ITheme.ts) and returns an [ITheme](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/interfaces/ITheme.ts).
```
Comment (joschect) [inline packages/variants/README.md]: Add a little something about how an ITheme will have default values not specified by IPartialTheme.

File: packages/variants/README.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
 The available variants are:
 
-* Neutral - a soft neutral color is used as the background (light gray in the default theme), most other colors are changed very little or not at all
-* Soft - a soft shade of the primary color is used as the background (light blue in the default theme), most other colors are changed very little or not at all, but input controls will look different
+* Neutral (NYI) - a soft neutral color is used as the background (light gray in the default theme), most other colors are changed very little or not at all
```
Comment (joschect) [inline packages/variants/README.md]: Whats NYI?

File: packages/variants/README.md [hunk 3]
```diff
@@ -1,11 +1,15 @@
 # [variants](http://dev.office.com/fabric)
 
-Variants are themes based on another theme, subthemes which share the same colors as the original theme, but with those colors used in different places. A variant is still a full theme in and of itself.
+Variants are [themes](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/ghdocs/Theming.md) generated from an existing theme (as opposed to [from raw colors](https://developer.microsoft.com/en-us/fabric#/styles/themegenerator)). A variant will share the same colors as the original theme it was generated from, but will use those colors differently. For example, the background color and text color might be swapped. Variants can be used to highlight or de-emphasize portions of the page.
```
Comment (joschect) [inline packages/variants/README.md]: You don't need parens, just add a comma so something like `existing theme, as opposed to ...`

File: packages/variants/README.md (hunk 1 above)
Comment (phkuo) [reply packages/variants/README.md]: isn't that implied by the interface definitions (and the names of the interfaces themselves)?

File: packages/variants/README.md (hunk 2 above)
Comment (phkuo) [reply packages/variants/README.md]: common TLA for not yet implemented

File: packages/variants/README.md (hunk 3 above)
Comment (phkuo) [reply packages/variants/README.md]: but i (REALLY) like parens!!

**Pull request merged**